coding-guides/making-your-own-starboard.md

```diff
@@ -123,7 +123,8 @@ module.exports = class {
     if (message.author.id === user.id) return message.channel.send(`${user}, you cannot star your own messages.`);
     if (message.author.bot) return message.channel.send(`${user}, you cannot star bot messages.`);
     const { starboardChannel } = this.client.settings.get(message.guild.id);
-    const fetch = await message.guild.channels.find('name', starboard).fetchMessages({ limit: 100 });
+    if (!message.guild.channels.exists('name', starboardChannel)) return message.channel.send(`It appears that you do not have a \`${starboardChannel}\` channel.`); 
+    const fetch = await message.guild.channels.find('name', starboardChannel).fetchMessages({ limit: 100 });
```

Comment:
same here, better to use find(channel => channel.name === starboardChannel)

Also, for readability: 'fetch' is a really bad name for a channel

Reply:
well its the original code 
also its . const fetch = await message.guild.channels.find('name', starboardChannel)**.fetchMessages({ limit: 100 });**